modules/home-environment.nix

```diff
@@ -229,7 +229,30 @@ in
 
     home.sessionVariables = mkOption {
       default = {};
-      type = types.attrs;
+      type = types.submodule {
+        freeformType = with types;
+          lazyAttrsOf (oneOf [ package str int float ]);
```

Comment:
What is a `lazyAttrsOf` :thinking: ?

Reply:
If I understand correctly, `lazyAttrsOf` is used to delay the evaluation of the attribute set  to be used for conditional `optionalAttrs` and other such things where recursion would happen if `mkIf` and `mkMerge` were to be used.